brian2/monitors/spikemonitor.py

```diff
@@ -357,7 +357,7 @@ def all_values(self):
                 "them."
             )
         indices = self.i[:]
-        sort_indices = np.argsort(indices)
```

Comment:
BTW: This was always wrong (we need a stable sort here), but it only now got triggered with numpy 1.25 (and only on GitHub, not on my local machine).